File: 백준/Bronze/29738. Goodbye， Code Jam/Goodbye， Code Jam.cs
int cnt = int.Parse(Console.ReadLine());

for (int i = 0; i < cnt; i++)
{
    int score = int.Parse(Console.ReadLine());

    if(score <= 25)
        Console.WriteLine($"Case #{i + 1}: World Finals");

    else if(score <= 1000)
        Console.WriteLine($"Case #{i + 1}: Round 3");

    else if (score <= 4500)
        Console.WriteLine($"Case #{i + 1}: Round 2");

    else
        Console.WriteLine($"Case #{i + 1}: Round 1");
}

File: 백준/Bronze/29738. Goodbye， Code Jam/Goodbye， Code Jam.py
cnt = int(input())

for i in range(cnt):
    score = int(input())

    if score <= 25:
        print(f"Case #{i + 1}: World Finals")
    elif score <= 1000:
        print(f"Case #{i + 1}: Round 3")
    elif score <= 4500:
        print(f"Case #{i + 1}: Round 2")
    else:
        print(f"Case #{i + 1}: Round 1")
